pkg/service/restore/service_restore_integration_test.go

```diff
@@ -300,21 +301,45 @@ func TestRestoreGetTargetUnitsViewsIntegration(t *testing.T) {
 				t.Fatal(err)
 			}
 
+			var ignoreTarget []string
+			if checkAnyConstraint(t, h.Client, ">= 5.5, < 2000", ">= 2024.2, > 1000") {
+				ignoreTarget = []string{
+					"!system_auth.*",
+					"!system_distributed.service_levels",
+				}
+			}
+
 			if diff := cmp.Diff(goldenTarget, target,
 				cmpopts.SortSlices(func(a, b string) bool { return a < b }),
 				cmpopts.IgnoreUnexported(Target{}),
 				cmpopts.IgnoreFields(Target{}, "SnapshotTag"),
+				cmpopts.IgnoreSliceElements(func(v string) bool { return slices.Contains(ignoreTarget, v) }),
 			); diff != "" {
 				t.Fatal(tc.target, diff)
 			}
 
+			var ignoreUnits []string
+			if checkAnyConstraint(t, h.Client, "< 1000") {
+				ignoreUnits = append(ignoreUnits, "system_replicated_keys")
+			}
+			if checkAnyConstraint(t, h.Client, "< 2024.2") {
+				ignoreUnits = append(ignoreUnits, "dicts")
+			}
+			if checkAnyConstraint(t, h.Client, ">= 5.5, < 2000", ">= 2024.2, > 1000") {
```

Comment:
As per my knowledge, 5.5 never went out (please correct me if I'm wrong). It can be >= 6.0

Reply:
I never did, but scylla-nightly had version 5.5 for some time and this test constraint was written so that it works fine with that. I will change it to 6.0 and see how the tests behave.